Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,209 @@
+          pradeo.detection.status: "{{json_event.message.content.event.kind}}"
+          pradeo.detection.value: "{{json_event.message.content.event.value}}"
+  correlation:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceCorrelationUpdated"}}'
+        set:
+          event.category: "process"
+          event.type: "change"
+          pradeo.device.id: "{{json_event.message.content.deviceCorrelation.device.id}}"
+          pradeo.device.serialNumber: "{{json_event.message.content.deviceCorrelation.device.serialNumber}}"
+          pradeo.device.name: "{{json_event.message.content.deviceCorrelation.device.name}}"
+          pradeo.device.email: "{{json_event.message.content.deviceCorrelation.device.email}}"
+          pradeo.device.imei: "{{json_event.message.content.deviceCorrelation.device.imei}}"
+          pradeo.device.byod: "{{json_event.message.content.deviceCorrelation.device.byod}}"
+          pradeo.device.declaredOperatingSystem: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystem}}"
+          pradeo.device.declaredOperatingSystemVersion: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemVersion}}"
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemSecurityPatchDate}}"
+          pradeo.device.declaredModel: "{{json_event.message.content.deviceCorrelation.device.declaredModel}}"
+          pradeo.device.lastConnection: "{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.lastConnection}}"
+          pradeo.device.coupled: "{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.coupled}}"
+          pradeo.device.mdmId: "{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.externalId}}"
+          pradeo.device.emm: "{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.emm}}"
+          pradeo.policy.id: "{{json_event.message.content.deviceCorrelation.detectionPolicy.id}}"
+          pradeo.policy.name: "{{json_event.message.content.deviceCorrelation.detectionPolicy.name}}"
+          pradeo.correlation.applicationThreatLevel: "{{json_event.message.content.deviceCorrelation.applicationThreatLevel}}"
+          pradeo.correlation.networkThreatLevel: "{{json_event.message.content.deviceCorrelation.networkThreatLevel}}"
+          pradeo.correlation.systemThreatLevel: "{{json_event.message.content.deviceCorrelation.systemThreatLevel}}"
+          pradeo.correlation.matchedSystemStatusLevels: "{{json_event.message.content.deviceCorrelation.matchedSystemStatusLevels}}"
+          pradeo.correlation.matchedNetworkStatusLevels: "{{json_event.message.content.deviceCorrelation.matchedNetworkStatusLevels}}"
+  compliance:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceApplicationComplianceUpdated"}}'
+        set:
+          event.category: "process"
+          event.type: "change"
```

These fields must be arrays
```suggestion
          event.category: ["process"]
          event.type: ["change"]
```